Templates/GenerateTestValues.tpl

```diff
@@ -102,15 +100,15 @@ namespace Services.Test.GenerateValues
 ;//
   <ALTERNATE_KEY_LOOP_UNIQUE>
     <SEGMENT_LOOP>
-            TestConstants.Instance.Get<StructureNoplural>_ByAltKey_<KeyName>_<SegmentName> = <IF DATEORTIME>DateTime.ParseExact((string)(<structureNoplural>.<segment_name>), "yyyyMMdd", System.Globalization.CultureInfo.InvariantCulture)<ELSE><IF SEG_TYPE_FIELD><structureNoplural>.<segment_name><ELSE SEG_TYPE_LITERAL>"<SEGMENT_LITVAL>"</IF SEG_TYPE_FIELD></IF DATEORTIME>
+            TestConstants.Instance.Get<StructureNoplural>_ByAltKey_<KeyName>_<SegmentName> = <IF DATEORTIME>DateTime.ParseExact((string)(<structureNoplural>.<segment_name>), <IF DATE>"yyyyMMdd"<ELSE>"hhmmss"</IF DATE>, System.Globalization.CultureInfo.InvariantCulture)<ELSE><IF SEG_TYPE_FIELD><structureNoplural>.<segment_name><ELSE SEG_TYPE_LITERAL>"<SEGMENT_LITVAL>"</IF SEG_TYPE_FIELD></IF DATEORTIME>
```

Comment:
There's a ton of date and time format strings, I don't think we can get away with hard coding just 1 here. see date conversion routine here https://github.com/Synergex/HarmonyCore/blob/d2cf0978d156b831912d8c37224e99fb99903a65/HarmonyCore/Converters/CustomDateRoutines.dbl#L117

Reply:
Fixed with commit f21ca12